media_kit_core_video/windows/angle_surface_manager.cc

```diff
@@ -43,6 +43,7 @@ HANDLE ANGLESurfaceManager::HandleResize(int32_t width, int32_t height) {
   }
   width_ = width;
   height_ = height;
+  glFinish();
```

Comment:
If you don't mind, can I review this out?

How about we keep all `glFinish` calls behind a macro in `angle_surface_manager.cc` like

```c

...

#define SAFEGUARD_GLFINISH

...

#ifdef SAFEGUARD_GLFINISH
  glFinish();
#endif

```

Reply:
sounds good to me, this one I don't even think is necessary, I only added it since it shouldn't be called that much and iirc remeber reading that someone had flicker issue on resize

Reply:
yeah I don't see much of a benefit to keeping this one, all it does is make it feel less sluggish, I couldn't observe any flickering on resize.